core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -159,6 +159,7 @@ class LogCleaner(initialConfig: CleanerConfig,
       cleaners += cleaner
       cleaner.start()
     }
+    activateMetrics();
```

Comment:
Would be useful to add the test in the original JIRA for posterity.

Reply:
+1 Please include a test

Reply:
Since there is no change to the existing definition of metrics, it seems this will cause the metrics to be initialized twice.